client/src/main/java/org/spine3/base/Commands.java

```diff
@@ -251,4 +250,44 @@ public static boolean isScheduled(Command command) {
         }
         return false;
     }
+
+    /**
+     * Sets a new scheduling time to {@link CommandContext.Schedule}.
+     *
+     * @param command a command to update
+     * @param schedulingTime a new scheduling time
+     * @return an updated command
+     */
+    public static Command setSchedulingTime(Command command, Timestamp schedulingTime) {
+        final Duration delay = command.getContext()
+                                      .getSchedule()
+                                      .getDelay();
+        final Command result = setSchedule(command, delay, schedulingTime);
+        return result;
+    }
+
+    /**
+     * Updates {@link CommandContext.Schedule}.
+     *
+     * @param command a command to update
+     * @param delay a delay to set
+     * @param schedulingTime a scheduling time to set
+     * @return an updated command
+     */
+    public static Command setSchedule(Command command, Duration delay, Timestamp schedulingTime) {
```

Comment:
Having to parameters `delay` and `schedulingTime` makes this method really puzzling. The documentation on the parameters doesn't add any value. It simply repeats the name of the parameters.